installers/linux-shared/after-remove.sh.erb

```diff
@@ -17,9 +17,9 @@
 -%>
 
 <% if type == 'rpm' -%>
-    rm -rf /var/lib/<%= name %> /var/log/<%= name %> /etc/defaults/<%= name %>
+    rm -rf /var/lib/<%= name %> /var/log/<%= name %>
 <% elsif type == 'deb' -%>
     update-rc.d <%= name %> remove
-    rm -rf /var/lib/<%= name %> /var/log/<%= name %> /etc/defaults/<%= name %>
+    rm -rf /var/lib/<%= name %> /var/log/<%= name %>
```

Comment:
It is there in the postuninstall script. So, it is called everytime an uninstall happens and not just on purge. @ketan told me "dirs are only removed by the package manager, if they are empty
so it’ll attempt to remove files first, then dirs if they are empty
if they are non-empty, it’s left to the discretion of the post uninstall script to do whatever it feels like"